openapi-cli/src/test/resources/expected/petstore.bal

```diff
@@ -3,10 +3,27 @@ import ballerina/http;
 listener http:Listener ep0 = new (80, config = {host: "petstore.openapi.io"});
 
 service /v1 on ep0 {
+    # List all pets
+    #
+    # + 'limit - How many items to return at one time (max 100)
+    # + return - returns can be any of following types
+    # Pets (An paged array of pets)
```

Comment:
@NipunaRanasinghe @lnash94 Due to the issue https://github.com/ballerina-platform/ballerina-lang/issues/29607, this multi line return description is not formatted correctly.

Reply:
Ack